docs.md

```diff
@@ -376,6 +376,21 @@ includes = ["my_great_lib.h"]
 # default: false
 no_includes = false
 
+# Whether to make a C header C++ compatible.
+# These will wrap generated functions into a `extern "C"` block, e.g.
```

Comment:
It also has effects on enums / enum classes and such. But this is a good start too.